rpc/shared/shared/src/main/scala/io/udash/rpc/annotations.scala

```diff
@@ -0,0 +1,4 @@
+package io.udash.rpc
+
+class RPC extends com.avsystem.commons.rpc.RPC
```

Comment:
I want these annotations to import with `io.udash.rpc._`. Importing `com.avsystem.commons.rpc.RPC` might be confusing.